tests/purs/failing/ClassHeadNoVTA3.out

```diff
@@ -0,0 +1,23 @@
+Error found:
+in module [33mMain[0m
+at tests/purs/failing/ClassHeadNoVTA3.purs:8:16 - 8:36 (line 8, column 16 - line 8, column 36)
+
+  No type class instance was found for
+  [33m                       [0m
+  [33m  Main.MultiMissing Int[0m
+  [33m                    t0 [0m
+  [33m                       [0m
+  The instance head contains unknown type variables. Consider using visible type application(s):
+  [33m                             [0m
+  [33m  useMultiMissing @norThisOne[0m
```

Comment:
This output still needs to be improved to something like `useMultiMissing @Int @norThisOne` because `@Int` is already being used.